core/jazz_ui/protractor.conf.js

```diff
@@ -4,6 +4,7 @@
 const SpecReporter = require('jasmine-spec-reporter').SpecReporter;
 var Jasmine2HtmlReporter = require('protractor-jasmine2-html-reporter');
 const JasmineConsoleReporter = require('jasmine-console-reporter');
+var JSONReporter = require('jasmine-json-test-reporter');
```

Comment:
please add the package to package.json if necessary.

Reply:
Added it on package json, but this is an import statement.